components/shared-image-gallery/variables.tf

```diff
@@ -25,4 +25,24 @@ variable "image_gallery_name" {
 
 variable "image_name" {
   default = "bastion-ubuntu"
+}
+
+variable "images" {
+  type = map(any)
+  default = {
+    "jenkins-agent" = {
+      "name"         = "jenkins-agent",
+      "managed_disk" = "jenkins-agent-disk",
+      "snapshot"     = "jenkins-agent-snapshot",
```

Comment:
a user should never need to specify this.

Reply:
Simplified to just key and sku